src/lib/swapper/swappers/ThorchainSwapper/getThorTradeQuote/getTradeQuote.ts

```diff
@@ -127,18 +118,38 @@ export const getThorTradeQuote = async (
       }),
     )
 
+  const getStreamingInterval = (depthBps: BigNumber): number => {
+    // Low health for the pools of this swap - use a longer streaming interval
+    if (depthBps.lt(5000)) return 5
+    // Moderate health for the pools of this swap - use a moderate streaming interval
+    if (depthBps.lt(9000) && depthBps.gte(5000)) return 3
+    return 1
+  }
+
   const streamingInterval =
     sellAssetPool && buyAssetPool
       ? (() => {
           const sellAssetDepthBps = sellAssetPool.derived_depth_bps
           const buyAssetDepthBps = buyAssetPool.derived_depth_bps
-          const swapDepthBps = bn(sellAssetDepthBps).plus(buyAssetDepthBps).div(2)
-          // Low health for the pools of this swap - use a longer streaming interval
-          if (swapDepthBps.lt(5000)) return 10
-          // Moderate health for the pools of this swap - use a moderate streaming interval
-          if (swapDepthBps.lt(9000) && swapDepthBps.gte(5000)) return 5
-          // Pool is at 90%+ health - use a 1 block streaming interval
-          return 1
+
+          // We are trading between 2 L1s, use a streaming interval of 1 block
```

Comment:
```suggestion
          // We are trading between 2 derived pools, use a streaming interval of 1 block
```

As L1s *do* have depth bps. Though updating the comment would effectively show that our logic is wrong here, as discussed on Discord:

```
Whether we "use" (very much quote on quote, see below) the old or new heuristics is irrelevant in this particular case, we would still get it wrong.
I believe the main issue here is we're not able to reliably get health for derived pools, and that rugs our whole logic
```